src/multiple_choice/card/css.css

```diff
@@ -21,6 +21,8 @@ table {
 }
 
 label {
+  display: inline-block;
+  vertical-align:middle;
```

Comment:
Why no space 😜? Just format the whole doc (if there are other missing spaces for a common formatting).

Reply:
Sorry, I don't get your point here. 🤔
Where are spaces missing?